pkg/kubectl/cmd/config/create_cluster.go

```diff
@@ -124,6 +126,9 @@ func (o *createClusterOptions) modifyCluster(existingCluster clientcmdapi.Cluste
 	if o.server.Provided() {
 		modifiedCluster.Server = o.server.Value()
 	}
+	if len(o.servers) > 0 {
```

Comment:
I remember that during initial discussion we agreed that both values will be used if provided, so if user will specify both server and servers we will create array [server, servers...] without duplications